packages/magento-customer/package.json

```diff
@@ -27,7 +27,8 @@
     "@graphcommerce/magento-graphql": "3.1.8",
     "@graphcommerce/magento-store": "4.2.32",
     "@graphcommerce/next-ui": "4.23.1",
-    "@graphcommerce/react-hook-form": "3.3.2"
+    "@graphcommerce/react-hook-form": "3.3.2",
+    "apollo-cache-persist": "0.1.1"
```

Comment:
The package is named `apollo3-cache-persist`, I believe you've got an old version